indico/modules/events/surveys/forms.py

```diff
@@ -43,6 +43,21 @@ class SurveyForm(IndicoForm):
     submission_limit = IntegerField(_("Capacity"),
                                     [HiddenUnless('limit_submissions'), DataRequired(), NumberRange(min=1)],
                                     description=_("Maximum number of submissions accepted"))
+    notifications_enabled = BooleanField(_('Notifications'), widget=SwitchWidget(),
```

Comment:
i think it is ok to have it so we don't clutter the form with unnecessary stuff - tanks to `HiddenUnless`